src/modules/ens/ens.service.ts

```diff
@@ -103,7 +105,7 @@ export class EnsService implements OnModuleDestroy {
       ensRegistryAddress: this.ensRegistry.address,
       provider: this.provider,
       domainNotifierAddress: DOMAIN_NOTIFIER_ADDRESS,
-      publicResolverAddress: PUBLIC_RESOLVER_ADDRESS,
```

Comment:
Ok, sorry, maybe I'm missing something in this change. Was `publicResolverAddress` renamed to `resolverAddress` as a part of this PR?

Reply:
No really. This renaming is part of change in https://github.com/energywebfoundation/ew-credentials/pull/18. But since we haven't came to conclusion about changing public resolver on resolver v2 in `getDomainsUsingResolver` probably we don't need to change resolver in this PR. I'll discard this